101-bakup-vm-recovery-services-vault/README.md

```diff
@@ -0,0 +1,16 @@
+# Backup Resource Manager IaaS VMs to Recovery Services Vault
```

Comment:
You may want to specify Backup Resource Manager IaaS VM (instead of IaaS VMs).
May also include another (older) template link to backup multiple iaas vms...